### PR TITLE
docs: update header, nav, landing page + add examples + deleting unused files

### PR DESCRIPTION
**Pull request by dgmccart**
- Updated the header to link back to the Portal site 
- Removed sidebar blue box link to docs site homepage 
- Deleted unused files
- Added requirements for installation to the quick start page
- Updated landing page copy and organization
- Adds example code snippets from @uermel 

Changes can be viewed in my branch of the docs site here: https://dgmccart.github.io/cryoet-data-portal/index.html

### CODE REVIEW

File: docs/cryoet_data_portal_docsite_FAQ.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # FAQ
 **To be updated after release**
+.. collapse:: Details
 
+    Example Text to see if the extension worked properly.
```
Comment (manasaV3) [inline docs/cryoet_data_portal_docsite_FAQ.md:5]: I believe this might need to updated. :)

File: docs/cryoet_data_portal_docsite_quick_start.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ This page provides details to start using the CryoET Data Portal.
 
 ## Installation
 
+### Requirements
+
+The CryoET Data Portal Client requires a Linux or MacOS system with:
+
+- Python 3.7 to Python 3.10. Or R, supported versions TBD.
```
Comment (manasaV3) [inline docs/cryoet_data_portal_docsite_quick_start.md:16]: I believe we should be supporting python 3.11 as well. Also, the client currently is only for python. We don't support R.

File: docs/cryoet_data_portal_docsite_quick_start.md [hunk 3]
```diff
@@ -17,10 +27,10 @@ python -m venv ./venv
 source ./venv/bin/activate
 ```
 
-Install the latest `cryoet-data-portal` package via pip:
+Install the latest `cryoet_data_portal` package via pip:
 
 ```shell
-pip install -U cryoet-data-portal
+pip install -U cryoet_data_portal
```
Comment (manasaV3) [inline docs/cryoet_data_portal_docsite_quick_start.md:33]: I believe this should be `pip install cryoet-data-portal`.

File: docs/cryoet_data_portal_docsite_FAQ.md (hunk 1 above)
Comment (dgmccart) [reply docs/cryoet_data_portal_docsite_FAQ.md]: Whoops, I didn't realize adding more commits after opening the PR would incorporate as well.

File: docs/cryoet_data_portal_docsite_quick_start.md (hunk 3 above)
Comment (dgmccart) [reply docs/cryoet_data_portal_docsite_quick_start.md]: Updated.

File: docs/cryoet_data_portal_docsite_quick_start.md (hunk 2 above)
Comment (dgmccart) [reply docs/cryoet_data_portal_docsite_quick_start.md]: Updated.

File: docs/cryoet_data_portal_docsite_quick_start.md [hunk 4]
```diff
@@ -17,15 +27,15 @@ python -m venv ./venv
 source ./venv/bin/activate
 ```
 
-Install the latest `cryoet-data-portal` package via pip:
+Install the latest `cryoet_data_portal` package via pip:
 
 ```shell
-pip install -U cryoet-data-portal
+pip install cryoet_data_portal
```
Comment (manasaV3) [inline docs/cryoet_data_portal_docsite_quick_start.md:33]: I believe this got missed. Can we have this be the same as before `pip install -U cryoet-data-portal` ?

**Review by manasaV3 — CHANGES_REQUESTED** [review]

**Review by manasaV3 — COMMENTED** [review]
There was a bug in an old version that required adding the list wrapper, it has been fixed. So, we can make the code more concise by removing the list wrapper.

**Comment by uermel** [comment]
Manasa is correct, I was still using the list wrapper for safety due to the old client version. I will fix the gist files, too.

File: docs/cryoet_data_portal_docsite_quick_start.md (hunk 4 above)
Comment (dgmccart) [reply docs/cryoet_data_portal_docsite_quick_start.md]: I switched it back.

**Comment by dgmccart** [comment]
> There was a bug in an old version that required adding the list wrapper, it has been fixed. So, we can make the code more concise by removing the list wrapper.

Thank you @manasaV3! I incorporated all of your code.

**Review by manasaV3 — APPROVED** [review]
lgmt!

**Pull request merged**